assets/wizards/analytics/views/custom-events/index.js

```diff
@@ -0,0 +1,280 @@
+
+/**
+ * Not implemented:
+ * - visibility, ini-load: require the element to be AMP element,
+ * - scroll: requires some more UI for scroll parameters, can be implemented later.
+ */
+const TRIGGER_OPTIONS = [
+	{ value: 'click', label: __( 'Click', 'newspack' ) },
+	{ value: 'submit', label: __( 'Submit', 'newspack' ) },
+];
+
+const NEW_EVENT_TEMPLATE = {
+	event_name: '',
+	event_category: '',
+	event_label: '',
+	on: TRIGGER_OPTIONS[ 0 ].value,
+	element: '',
+	amp_element: '',
+	non_interaction: true,
+	is_active: true,
+};
+
+const validateEvent = event =>
+	Boolean( event.event_name && event.event_category && event.on && event.element );
+
+/**
+ * Analytics Custom Events screen.
+ */
+class CustomEvents extends Component {
+	state = {
+		error: newspack_analytics_wizard_data.analyticsConnectionError,
+		customEvents: newspack_analytics_wizard_data.customEvents,
+		editedEvent: NEW_EVENT_TEMPLATE,
+		editedEventId: null,
+	};
+
+	handleAPIError = ( { message: error } ) => this.setState( { error } );
+
+	updateCustomEvents = updatedEvents => {
+		const { wizardApiFetch } = this.props;
+		wizardApiFetch( {
+			path: '/newspack/v1/wizard/analytics/custom-events',
+			method: 'POST',
+			data: { events: updatedEvents },
+		} )
+			.then( ( { events } ) =>
+				this.setState( {
+					customEvents: events,
+					editedEvent: NEW_EVENT_TEMPLATE,
+					editedEventId: null,
+				} )
+			)
+			.catch( this.handleAPIError );
+	};
+
+	handleCustomEventEdit = () => {
+		const { customEvents, editedEvent, editedEventId } = this.state;
+		if ( editedEventId === 'new' ) {
+			this.updateCustomEvents( [ ...customEvents, editedEvent ] );
+		} else {
+			this.updateCustomEvents(
+				customEvents.map( event => {
+					if ( event.id === editedEventId ) {
+						return editedEvent;
+					}
+					return event;
+				} )
+			);
+		}
+	};
+
+	updateEditedEvent = key => value =>
+		this.setState( ( { editedEvent } ) => ( { editedEvent: { ...editedEvent, [ key ]: value } } ) );
+
+	setEditModal = editedEventId => () => {
+		const editedEvent =
+			editedEventId !== null && find( this.state.customEvents, [ 'id', editedEventId ] );
+		this.setState( {
+			editedEventId,
+			...( editedEvent
+				? {
+						editedEvent,
+				  }
+				: {
+						editedEvent: NEW_EVENT_TEMPLATE,
+				  } ),
+		} );
+	};
+
+	render() {
+		const { error, customEvents, editedEvent, editedEventId } = this.state;
+		const { isLoading } = this.props;
+
+		const isCreatingEvent = editedEventId === 'new';
+
+		return (
+			<div className="newspack__analytics-configuration newspack-card newspack-card__no-background">
+				<p>
+					{ __(
+						'Custom events are used to collect and analyze specific user interactions.',
+						'newspack'
+					) }
+				</p>
+				<Notice
+					rawHTML
+					isInfo
+					noticeText={ `${ __(
+						'This is an advanced feature, read more about it on our',
+						'newspack'
+					) } <a href="${ NEWSPACK_SUPPORT_URL }/analytics">${ __(
```

Comment:
Another really minor nitpick: 

```suggestion
					) } <a target="_blank" href="${ NEWSPACK_SUPPORT_URL }/analytics">${ __(
```

Reply:
Done in 5ae9711750d7e43a80fd15b32bcfed700c7e849b

Reply:
Thanks Adam!